FreeRTOS_ARP.c

```diff
@@ -725,17 +725,16 @@ void FreeRTOS_OutputARPRequest( uint32_t ulIPAddress )
 
         #if defined( ipconfigETHERNET_MINIMUM_PACKET_BYTES )
             {
-                if( pxNetworkBuffer->xDataLength < ( size_t ) ipconfigETHERNET_MINIMUM_PACKET_BYTES )
```

Comment:
We talked about this if() clause: it must stay there. You have corrected that in the PR for FreeRTOS_ARP.c, and the change shouldn't be here.